### PR TITLE
[RM Ch3] HW Acceleration Resources

### PR DESCRIPTION
**Pull request by petorre**
Towards fixing #2051 .

### CODE REVIEW

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -222,9 +222,18 @@ The physical compute, storage and network resources serve as the foundation of t
...
+
+| Reason for using Hardware Acceleration | Example | Comment |
+|---|---|---|
+| Achieve technical requirements | Strict latency or timing accuracy | Must be done by optimizing compute node; cannot be solved by adding more compute nodes |
+| Achieve technical requirements | Fit within power or space envelope | Done by optimizing cluster of compute nodes |
+| Improve cost/performance | Better cost and less power/cooling by improving performance per node | Used when functionality can be achieved through usage of accelerator or by adding more compute nodes |
+
+Hardware Accelerators can be used to offload software execution for purpose of accelerating tasks to achieve more or faster performance, or offloading them to another execution entity to get more predictable execution times, efficient handling of the tasks or separation of authority regarding who can control the tasks execution.
+
+More details about Hardware Acceleration are in [Section 3.8 Hardware Acceleration Abstraction](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter03.md#3.8).
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter03.md:235]:
```suggestion
More details about Hardware Acceleration are in [Section 3.8 Hardware Acceleration Abstraction](chapter03.md#3.8).
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter03.md [hunk 2]
```diff
@@ -222,9 +222,18 @@ The physical compute, storage and network resources serve as the foundation of t
...
-...
+For a given software network function and software infrastructure, Hardware Acceleration resources can be used to achieve requirements or improve cost/performance. Following table gives reasons and examples for using Hardware Acceleration.
+
+| Reason for using Hardware Acceleration | Example | Comment |
+|---|---|---|
+| Achieve technical requirements | Strict latency or timing accuracy | Must be done by optimizing compute node; cannot be solved by adding more compute nodes |
+| Achieve technical requirements | Fit within power or space envelope | Done by optimizing cluster of compute nodes |
+| Improve cost/performance | Better cost and less power/cooling by improving performance per node | Used when functionality can be achieved through usage of accelerator or by adding more compute nodes |
+
+Hardware Accelerators can be used to offload software execution for purpose of accelerating tasks to achieve more or faster performance, or offloading them to another execution entity to get more predictable execution times, efficient handling of the tasks or separation of authority regarding who can control the tasks execution.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.md:233]:
```suggestion
Hardware Accelerators can be used to offload software execution for purpose of accelerating tasks to achieve faster performance, or offloading the tasks to another execution entity to get more predictable execution times, efficient handling of the tasks or separation of authority regarding who can control the tasks execution.
```

**Comment by walterkozlowski** [comment]
@petorre: I am approving but please make sure that the diagram is updated

**Review by walterkozlowski — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Review by TFredberg — APPROVED** [review]

**Comment by petorre** [comment]
Added table caption as per agreement on RM call.

**Pull request merged**